pkg/keyspace/tso_keyspace_group.go

```diff
@@ -792,10 +792,10 @@ func (m *GroupManager) AllocNodesForKeyspaceGroup(id uint32, desiredReplicaCount
 			if addr == "" {
```

Comment:
Do we need to check whether tso node is available here?

Reply:
The node balancer will be updated through the watch event.

Reply:
When a watch event is delayed, an unavailable tso is allocated. Maybe we can add an available check?